element_array_ephys/version.py

```diff
@@ -1,2 +1,2 @@
 """Package metadata."""
-__version__ = "0.2.1"
+__version__ = "0.2.0"
```

Comment:
Changing here to reflect CHANGELOG, otherwise looks good
```suggestion
__version__ = "0.2.2"
```